lib/commands/doctor/checks/install.js

```diff
@@ -68,6 +68,19 @@ const tasks = {
             return tasks.checkDirectoryAndAbove(process.cwd(), 'run `ghost install`');
         });
     },
+    currentUser: function currentUser() {
+        // Check if user is trying to install as `root`
+        if (process.getuid() === 0) {
+            return Promise.reject(new errors.SystemError(
+                `Can't install as 'root' user.${eol}` +
+                `Please create a new user with regular account privileges to install Ghost.${eol}` +
+                `See ${chalk.underline.blue('https://docs.ghost.org/docs/install')} ` +
```

Comment:
Do we want to link to the specific section of the page that talks about the non-root user or just to the page itself?

Reply:
Yeah, was thinking to do that but than thought that's it might be to volatile. But you're right... I'll change it to link to the section.